### PR TITLE
KOGITO-3407 CloudEvent Kafka Publisher+Emitter (Spring Boot)

### PR DESCRIPTION
**Pull request by evacchi**
https://issues.redhat.com/browse/KOGITO-3407

related https://github.com/kiegroup/kogito-runtimes/pull/839

----

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by danielezonca** [comment]
@evacchi 
Can you please merge master? I still see Quarkus changes in files diff

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

**Review by radtriste — APPROVED** [review]

**Review by tiagodolphine — APPROVED** [review]
LGTM, just a curiosity, in case we need to subscribe or publish to more than one topic, this parameter is parsed as a list?

File: process-kafka-quickstart-springboot/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-
 spring.kafka.consumer.group-id=travellers-group
-spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
-spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
-
-spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
-spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
-spring.kafka.producer.properties.spring.json.add.type.headers=false
+kogito.addon.cloudevents.kafka.kogito_incoming_stream=travellers
+kogito.addon.cloudevents.kafka.kogito_outgoing_stream=processedtravellers
```
Comment (ricardozanini) [inline process-kafka-quickstart-springboot/src/main/resources/application.properties:5]: Maybe these properties should be the same for Quarkus and Spring? Or at least in Quarkus we could recognize them as well...

**Review by ricardozanini — APPROVED** [review]

File: process-kafka-quickstart-springboot/src/main/resources/application.properties (hunk 1 above)
Comment (evacchi) [reply process-kafka-quickstart-springboot/src/main/resources/application.properties:5]: yes we should definitely do something there... https://issues.redhat.com/browse/KOGITO-3737

**Pull request merged**